File: Basic/code/problem_26.py
'''
Write a program to Print the following pattern:
*
* *
* * *
* * * *
'''

# Solved by Ashani2529
n=4
def pypart(n):
	for i in range(0, n):
		for j in range(0, i+1):
			print("* ",end="")
		print("\r")
pypart(n)

File: Basic/code/problem_26.c
/*26. Write a program to print the following pattern:
* 
* * 
* * * 
* * * *    */

//Solved by VibhowSingh56
#include <stdio.h>
int main() {
   int i, j;
   for (i = 1; i <= 4; ++i) {
      for (j = 1; j <= i; ++j) {
         printf("* ");
      }
      printf("\n");
   }}
